solr/solrj/src/test/org/apache/solr/client/solrj/embedded/LargeVolumeJettyTest.java

```diff
@@ -16,12 +16,17 @@
  */
 package org.apache.solr.client.solrj.embedded;
 
+import org.apache.lucene.tests.util.LuceneTestCase;
 import org.apache.solr.client.solrj.LargeVolumeTestBase;
+import org.apache.solr.util.ExternalPaths;
 import org.junit.BeforeClass;
 
 public class LargeVolumeJettyTest extends LargeVolumeTestBase {
   @BeforeClass
   public static void beforeTest() throws Exception {
-    initCore();
+    // TODO
```

Comment:
left over or real?

Reply:
See previous thread I at-mentioned you on.